aquacomputer_d5next.c

```diff
@@ -831,7 +840,21 @@ static umode_t aqc_is_visible(const void *data, enum hwmon_sensor_types type, u3
 				switch (attr) {
 				case hwmon_pwm_input:
 				case hwmon_pwm_mode:
-					return 0644;
+					/*
+					 * Wait until the first Aquaero sensor report is received,
+					 * to be able to differentiate between Aquaero 5 and 6
+					 * by looking at the hardware version. While the v6 supports
+					 * both DC and PWM mode for all four fans, v5 supports PWM
+					 * mode only for the fourth fan.
+					 */
+					if (!wait_for_completion_timeout
+					    (&aquaero_sensor_report_received,
+					     STATUS_UPDATE_INTERVAL))
+						return 0;
+
+					if (priv->aquaero_hw_kind == aquaero5 && channel == 3 ||
```

Comment:
Use parenthesis around the Aquaero 5 part as the compile error suggests.

Reply:
Pretty sure I did this locally... Anyway, fixed.